services/src/main/java/org/keycloak/exportimport/util/ExportUtils.java

```diff
@@ -71,11 +60,20 @@
 import org.keycloak.representations.idm.authorization.ScopeRepresentation;
 import org.keycloak.util.JsonSerialization;
 
-import com.fasterxml.jackson.core.JsonEncoding;
-import com.fasterxml.jackson.core.JsonFactory;
-import com.fasterxml.jackson.core.JsonGenerator;
-import com.fasterxml.jackson.databind.ObjectMapper;
-import com.fasterxml.jackson.databind.SerializationFeature;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
```

Comment:
@hmlnarik Do we have a template for the imports? Some files in the codebase have the java and javax packages first, others have them at the end of the imports list. Just wondering because IIRC we've been keeping java and javax at the top.

Reply:
Regardless of the case, we have always avoided introducing these import reorganizations in commits.

Reply:
I had the auto-format for imports enabled when I missed some unused imports the other day. Will revert this change, and re-calibrate my IDE. If there is some information how to configure IntelliJ, or some rules that I can translate for IntelliJ, I'd be happy to do so. Another rule I learned in another review: no star imports :roll_eyes:

Reply:
I don't think there is any strict rule for import ordering.
Generally the aim is to keep PRs minimal and focused on the particular task so the aim is to prevent large purely syntactical changes.